microsoft/testsuites/nvme/nvme.py

```diff
@@ -346,6 +346,12 @@ def nvme_manage_ns_validation(self, node: Node) -> None:
             format_namespace.assert_exit_code(device_format_exit_code)
             # 3. `nvme create-ns namespace` - create a namespace.
             create_namespace = nvme_cli.create_namespace(namespace)
+            # NVMe Status:INVALID_OPCODE: The associated command opcode field is not
```

Comment:
It's good to have comments, but it's better to explain why two different error code is here. And I don't understand how (0x1) translate to 22.

Reply:
post an issue in nvme-cli github to ask this question.